rake/spec/default/60-st2_all-services-ok_spec.rb

```diff
@@ -63,4 +66,21 @@
       it { is_expected.to be_running }
     end
   end
+
+  describe 'st2auth', prompt_on_failure: true do
+    subject { port(9100) }
+    it { should be_listening }
```

Comment:
I guess this is kind of Ruby-madness around "beauty/ugly" :)

```ruby
describe 'st2api', prompt_on_failure: true do
  subject { port(9101) }
  it { is_expected.to be_listening }
end
```
To be sure, ^^ this is what you expect, right?